packages/react-hooks/src/useThresholds.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017-2021 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { useEffect, useState } from 'react';
+
+import { getProposalThreshold,
+  getSlashProposalThreshold,
+  getTreasuryProposalThreshold,
+  getTreasuryRejectionThreshold } from '@polkadot/apps-config';
```

Comment:
These imports are not needed, as per the helpers can be included here.

Reply:
Done :)